docs/articles/sde/introduction_to_sde.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+Software Development Environment
+********************************
+
+PELUX Software Development Environment (SDE) is a tool that creates a virtual
+machine  with all the tools needed when developing towards a PELUX platform.
+More specifically the SDE is setup using vagrant to generate a VirtualBox
+instance. In order to facilitate this, the PELUX SDE requires an instance of
+PELUX SDK to be present while setting up the environment. The source code can
+be found at https://github.com/Pelagicore/PELUX-SDE. Please read the README.md
```

Comment:
Turn this link into a link instead, if you know what I mean

Reply:
like this?